content-src/components/Sections/_Sections.scss

```diff
@@ -84,11 +84,6 @@
       font-size: 13px;
     }
   }
-
-  @media (min-width: $break-point-large) {
-    line-height: 16px;
-    height: 16px;
```

Comment:
This does change `top-stories-bottom-container`'s height to 20.8px from 16px; is that intended?

Reply:
Well, we really shouldn't be setting fixed heights on anything containing live text unless there's a good reason to, so...yes?

Reply:
Why was height set in the first place?

Reply:
I *think* it was something to do with the height transition for collapsing the panel, it was causing some jitter and movement for the highlights container. Something I currently cannot reproduce.

Reply:
Other than that, I cannot really see what it's doing.

Reply:
Ok, I think it's probably safe to remove then.